.github/CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
-## Release 0.4.0 (development release)
+## Release 0.3.2 (development release)
```

Comment:
Sorry, one last thing: in order to release the XCC immediately, we should update these version annotations:
```suggestion
## Release 0.3.2 (current release)
```